target/scripts/startup/setup-stack.sh

```diff
@@ -668,18 +645,27 @@ function _setup_dkim_dmarc
     # setup nameservers parameter from /etc/resolv.conf if not defined
     if ! grep '^Nameservers' /etc/opendkim.conf
     then
-      echo "Nameservers $(grep '^nameserver' /etc/resolv.conf | awk -F " " '{print $2}' | paste -sd ',' -)" >>/etc/opendkim.conf
-
+      local NAMESERVER_IPS
+      NAMESERVER_IPS=$(grep '^nameserver' /etc/resolv.conf | awk -F " " '{print $2}' | paste -sd ',' -)
+      echo "Nameservers ${NAMESERVER_IPS}" >>/etc/opendkim.conf
       _log 'trace' "Nameservers added to '/etc/opendkim.conf'"
     fi
   fi
 
   if [[ ${ENABLE_OPENDMARC} -eq 1 ]]
   then
     _log 'trace' "Adding OpenDMARC to Postfix's milters"
-
+    # Make sure to append the milter _behind_ the OpenDKIM milter!
```

Comment:
Behind? After is probably more accurate? :)

```suggestion
    # Make sure to append the OpenDMARC milter _after_ the OpenDKIM milter!
```

Reply:
Yeah, you're right.. :D

Reply:
Done in [9756028](https://github.com/docker-mailserver/docker-mailserver/pull/3059/commits/97560282a7e640e80b9160df781366ec019744d0)

Reply:
The comment suggested change? It didn't seem to get applied :thinking:

Reply:
Oh my, sorry! I know why, a stupid mistake. Will push one more commit..